mswindows/osgeo4w/package.sh

```diff
@@ -111,7 +111,8 @@ read MINOR <&3
 read PATCH <&3
 
 export VERSION=${MAJOR}.${MINOR}.${PATCH}
-export POSTFIX=${MAJOR}${MINOR}
+#export POSTFIX=${MAJOR}${MINOR}
```

Comment:
we have this logic with ${MAJOR}${MINOR} that we're able to build/compile e.g. 7.6, 7.8, 7.9, 8.0 etc in the same OSGeo4W build environment side by side; IIRC also used later for compiling the winGRASS addons for different versions.

so if we decide not to use ${MAJOR}${MINOR}, then maybe OSGeo4W build environment has to be re-organized.

@landam what are your suggestions?